### PR TITLE
package & example updated with MUI 6

### PR DESCRIPTION
**Pull request by shubham-eaton**
#### Changes proposed in this Pull Request:

- Updated package and example with Mui 6
- Removed deprecated APIs
-

#### To Test:

- yarn start:example

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: login-workflow/example/src/index.tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import 'react-app-polyfill/ie11';
-import { ThemeProvider, createTheme, StyledEngineProvider } from '@mui/material/styles';
+import { ThemeProvider, StyledEngineProvider } from '@mui/material/styles';
 import 'react-app-polyfill/stable';
 import React from 'react';
 import * as serviceWorker from './serviceWorker';
 import { App } from './App';
 import CssBaseline from '@mui/material/CssBaseline';
-import * as BLUIThemes from '@brightlayer-ui/react-themes';
```
Comment (surajeaton) [inline login-workflow/example/src/index.tsx:8]: Please run 'yarn prettier'

File: login-workflow/package.json [hunk 2]
```diff
@@ -75,13 +76,13 @@
         "@babel/preset-typescript": "^7.14.5",
         "@brightlayer-ui/eslint-config": "^3.0.0",
         "@brightlayer-ui/prettier-config": "^1.0.3",
-        "@brightlayer-ui/react-components": "^6.1.2-alpha.0",
```
Comment (surajeaton) [inline login-workflow/package.json:78]: Please update @brightlayer-ui/react-themes to 8.0.0-alpha.0

File: login-workflow/package.json [hunk 3]
```diff
@@ -35,8 +36,8 @@
         "@brightlayer-ui/react-components": "^6.1.2",
```
Comment (surajeaton) [inline login-workflow/package.json:36]: Update react components

File: login-workflow/package.json (hunk 2 above)
Comment (shubham-eaton) [reply login-workflow/package.json:78]: changes done

File: login-workflow/package.json (hunk 3 above)
Comment (shubham-eaton) [reply login-workflow/package.json]: changes done

File: login-workflow/example/src/index.tsx (hunk 1 above)
Comment (shubham-eaton) [reply login-workflow/example/src/index.tsx:8]: changes done

**Review by surajeaton — APPROVED** [review]

**Pull request merged**